doc/sphinx/changes.rst

```diff
@@ -470,10 +470,13 @@ Commands
 - **Removed:** ``SearchAbout`` command that was deprecated since 8.5.
   Use :cmd:`Search` instead
   (`#11944 <https://github.com/coq/coq/pull/11944>`_, by Jim Fehrle).
-- **Deprecated:**
-  Declaration of arbitrary terms as hints. Global references are now
-  preferred (`#7791 <https://github.com/coq/coq/pull/7791>`_, by
-  Pierre-Marie Pédrot).
+- **Changed:**
+  Declaring a hint which is not a global reference now generates
+  a constant on the fly. This behaviour can be tweaked by
+  unsetting the deprecated :flag:`Declare Hint Proxy` flag
+  (`#12493 <https://github.com/coq/coq/pull/12493>`_,
+  fixes `#11970 <https://github.com/coq/coq/issues/11970>`_,
+  by Pierre-Marie Pédrot).
```

Comment:
A very small remark is that the "Changed" entry should be inserted above, between the last "Changed" and the first "Removed" entry.

And note that this still depends on getting accepted in 8.12+beta1. If it isn't, then this will have to be done differently.